test/unit/repositories/conftest.py

```diff
@@ -62,3 +64,23 @@ def mock_find_one(collection_mock: Mock, document: dict | None) -> None:
             documents = list(collection_mock.find_one.side_effect)
             documents.append(document)
             collection_mock.find_one.side_effect = documents
+
+    @staticmethod
+    def mock_find(collection_mock: Mock, documents: List[dict], query: Optional[dict] = None) -> None:
+        """
+        Mocks the `find` method of the MongoDB database collection mock to return a specific list of documents.
+
+        :param collection_mock: Mocked MongoDB database collection instance.
+        :param documents: The list of documents to be returned by the `find` method.
+
+        """
+        if query is None:
+            filtered_documents = documents
+        else:
+            filtered_documents = [
+                doc for doc in documents if all(query_param in doc.items() for query_param in query.items())
+            ]
```

Comment:
Filtering these isn't necessary here, the mock of the database query shouldn't care what query it is given. I think its worth keeping it consistent with IMS.